src/app/pages/access-denied-beta/access-denied-beta.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import { Title } from '@angular/platform-browser';
+
+@Component({
+  selector: 'app-access-denied-beta',
+  templateUrl: './access-denied-beta.component.html',
+  styleUrls: ['./access-denied-beta.component.css']
+})
+export class AccessDeniedBetaComponent implements OnInit {
+
+  constructor(private title: Title) { }
+
+  ngOnInit(): void {
+    this.title.setTitle('Access Denied (Not a Beta Tester) | Virtrolio');
```

Comment:
```suggestion
    this.title.setTitle('Beta Access Denied | Virtrolio');
```

Title a bit long to be readable